### ISSUE TITLE
 Label associated with the Select element is not read by screen reader

### ISSUE THREAD

**Issue by nagbhushanr** [SAP/fundamental-ngx#9088]
#### Is this a bug, enhancement, or feature request?
bug

#### Briefly describe your proposal.
Observed:
The header or Label associated with the Select element is not read by screen reader.
Expected:
Associated label to be read by screen reader

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
angular - 13.3.7
fundamental ngx - 0.36.0

#### If this is a bug, please provide steps for reproducing it.
https://635720a7e34ffc00087a288e--fundamental-ngx.netlify.app/#/core/select/api

#### Is there anything else we should know?
![image](https://user-images.githubusercontent.com/119582923/207274833-9fac62a7-d1df-4d1a-ac76-fb7aa273f999.png)

**Comment by N1XUS** [comment]
Hello @nagbhushanr.
Could you please provide a stackblitz example with the markup you're using?

**Comment by nagbhushanr** [comment]
https://stackblitz.com/edit/angular-4yz52b?file=src%2Fmain.ts,src%2Fapp%2Fselect-mode-example.component.html,src%2Fapp%2Fselect-mode-example.component.ts
Please use the "JAWS" tool to screen read if you are using windows or if you are using mac i believe it has a inbuilt tool for that.
Basic Idea is that the label associated with **Select** component should be read out by screen reader

**Comment by N1XUS** [comment]
Hello @nagbhushanr thank you for the example.
We provide options to set aria-labelled-by and aria-label:
https://stackblitz.com/edit/angular-4yz52b-ferkrn?file=src%2Fmain.ts,src%2Fapp%2Fselect-mode-example.component.html,src%2Fstyles.scss

You can check this example and adjust your code as shown
